Win32/Prj/libpcap.vcxproj.filters

```diff
@@ -19,9 +19,6 @@
     <ClCompile Include="..\..\gencode.c">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="..\..\grammar.c">
```

Comment:
Again, the build process should run Flex and Bison, which should put those source files in the main source directory, so that change shouldn't be necessary.